test/index.js

```diff
@@ -506,9 +820,18 @@ describe('def', function() {
 
     throws(function() { convertTo('days', new Date(0)); },
            errorEq(TypeError,
-                   '‘convertTo’ expected a value of type ' +
-                   '("milliseconds" | "seconds" | "minutes" | "hours") ' +
-                   'as its first argument; received "days"'));
+                   'Invalid value\n' +
+                   '\n' +
+                   'convertTo :: ("milliseconds" | "seconds" | "minutes" ' +
+                   '| "hours") -> ValidDate -> ValidNumber\n' +
+                   '             ' +
+                   '^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n' +
+                   '                                     1\n' +
```

Comment:
I suggest we ignore our maximum line length for these errors messages.

```javascript
                   'convertTo :: ("milliseconds" | "seconds" | "minutes" | "hours") -> ValidDate -> ValidNumber\n' +
                   '             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n' +
                   '                                     1\n' +
```

I'd like to be able to confirm visually that the correct characters are underlined (even if I must scroll to do so).